CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.7.1] - 2022-12-20
+## [Unreleased]
+
+## [1.8.2] - 2023-01-10
+
+### Fixed
+
+- Fixed an issue where inserting certain types of data would loop forever.
+
+## [1.8.1] - 2022-12-20
+
 ### Added
-- Add Artifactory authentication to Jenkinsfile
 
-## [Unreleased]
+- Add Artifactory authentication to Jenkinsfile
```

Comment:
I don't actually see any code be added in here for authentication.

Reply:
I think this was just me re-ordering change log entries since 1.7.1 was mistakenly added after 1.8.0 before.